packages/botonic-plugin-flow-builder/src/index.ts

```diff
@@ -61,7 +61,14 @@ export default class BotonicPluginFlowBuilder implements Plugin {
     })
   }
 
-  async post(_request: PluginPostRequest): Promise<void> {}
```

Comment:
isn't it mandatory to define the post?

Reply:
no in the latest version of botonic core the pre and post functions are optional, but you have to implement at least one in order to implement Plugin.